package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-source",
-  "version": "6.7.0-alpha.1",
+  "version": "6.7.0-alpha.1.nvp-asset-sizes+ad603139",
```

Comment:
?

Reply:
removed in latest push. happens when you run `pnpm build` or build-for-production -- i forget which